src/routes/user.routes.ts

```diff
@@ -188,11 +186,13 @@ router.post("/createadmin", async (req, res) => {
     isAdmin: true,
   };
 
-  const saved = await new User(admin).save();
-  if (!saved) {
+  const isTaken = await User.findById({ email: "admin@a.com" });
```

Comment:
I don't like hardcoded values, make it either const somewhere or use environmental variables